File: app/src/main/java/ru/melod1n/schedule/common/TaskManager.java
package ru.melod1n.schedule.common;


import ru.melod1n.schedule.concurrent.LowThread;

public class TaskManager {

    private static final String TAG = "TaskManager";

    public static void execute(Runnable runnable) {
        new LowThread(runnable).start();
    }

}

File: app/src/main/java/ru/melod1n/schedule/common/TaskManager.kt
package ru.melod1n.schedule.common

import ru.melod1n.schedule.concurrent.LowThread

object TaskManager {

    fun execute(runnable: Runnable) {
        LowThread(runnable).start()
    }

    fun execute(unit: () -> Unit) {
        execute(Runnable(unit))
    }

    fun executeNow(runnable: Runnable) {
        Thread(runnable).start()
    }

    fun executeNow(unit: () -> Unit) {
        executeNow(Runnable(unit))
    }

}